src/test/java/hill/weatherconfidence/ApplicationContextTest.java

```diff
@@ -5,11 +5,11 @@
 import org.springframework.context.annotation.Primary;
 
 @TestConfiguration
-public class TestApplicationContext {
+public class ApplicationContextTest {
 
     @Primary
     @Bean
     public IKmsDecrypter kmsDecrypter() {
-        return new TestKmsDecrypter();
+        return new DummyKmsDecrypter();
```

Comment:
We don't want to have to mock up the Google API behaviour, so we inject a dummy implementation of the interface for the tests.